meta-resin-common/recipes-connectivity/resin-proxy-config/resin-proxy-config/resin-proxy-config

```diff
@@ -54,4 +54,4 @@ iptables -t nat -A REDSOCKS -d 192.168.0.0/16 -j RETURN
 iptables -t nat -A REDSOCKS -d 224.0.0.0/4 -j RETURN
 iptables -t nat -A REDSOCKS -d 240.0.0.0/4 -j RETURN
 iptables -t nat -A REDSOCKS -p tcp -j REDIRECT --to-ports 12345
-iptables -t nat -A OUTPUT -p tcp -j REDSOCKS
+iptables -t nat -A PREROUTING -p tcp -j REDSOCKS
```

Comment:
I think we adapt when we delete the rules form the chain `iptables -t nat -D OUTPUT -p tcp -j REDSOCKS || true`.

Reply:
Good point, amended